pandas/tools/tile.py

```diff
@@ -81,6 +82,13 @@ def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
     array([1, 1, 1, 1, 1], dtype=int64)
     """
     # NOTE: this binning code is changed a bit from histogram for var(x) == 0
+    # for handling the cut for datetime and timedelta objects
+    if needs_i8_conversion(x):
+        x = x.values.view('i8')
+        time_data = True
```

Comment:
@jreback the x.dtype call errors out with this message when i run nosetests on test_tile.py AttributeError: 'list' object has no attribute 'dtype'. The formatter that i create in the suggested change here will be used in lieu of the formatter that is present in the downstream code, in method _format_levels  , is that correct?